CMakeLists.txt

```diff
@@ -42,6 +42,12 @@ option(S2_USE_SYSTEM_INCLUDES
 add_feature_info(S2_USE_SYSTEM_INCLUDES S2_USE_SYSTEM_INCLUDES
     "Silence warnings in s2 headers by marking them as SYSTEM includes.")
 
+if(NOT CMAKE_SOURCE_DIR STREQUAL PROJECT_SOURCE_DIR)
```

Comment:
Please add a comment (abseil-cpp had one), it is not at all obvious what this is doing here.